packages/paste-website/src/components/site-wrapper/SiteBody.tsx

```diff
@@ -14,22 +15,27 @@ const StyledSiteBody = styled.div`
 
   @supports (display: grid) {
     display: grid;
-    grid-template-columns: ${themeGet('sizes.sizeSidebar')} 1fr;
+    grid-template-columns: 1fr;
+
+    @media screen and (min-width: ${SITE_BREAKPOINTS[1]}) {
+      grid-template-columns: ${themeGet('sizes.sizeSidebar')} 1fr;
+    }
   }
 `;
 
 export const SiteBody: React.FC = ({children}) => {
+  const {breakpointIndex} = useWindowSize();
+  const isDesktop = breakpointIndex === undefined || breakpointIndex > 1;
+
   return (
     <>
       <SiteHeader />
       <StyledSiteBody>
-        <Sidebar />
-        <Box minWidth="size0">
-          <SiteMain role="main">
-            <ScrollAnchorIntoView />
-            <SiteMainInner>{children}</SiteMainInner>
-            <SiteFooter />
-          </SiteMain>
+        {isDesktop ? <Sidebar /> : null}
```

Comment:
I think it'd be better to just use the responsive display props here

Reply:
In this case I removed the `isDesktop` variable name and inlined the condition. Essentially I render it while `breakpointIndex` is still `undefined`, and use CSS within Sidebar to display it or not. Once the page is hydrated I conditionally render it.